### PR TITLE
Feature/currentcurrency

### PR DESCRIPTION
**Pull request by cristhianbenitez**
props drilling throught the app to display the selected currency symbol and values

### CODE REVIEW

File: src/components/AssetsListRow/AssetsListRow.js [hunk 1]
```diff
@@ -98,6 +98,10 @@ export class AssetsListRow extends Component {
       return isPositive ? <GreenArrowUp /> : <RedArrowDown />;
     };
 
+    const currency = `${this.props.currency}`;
```
Comment (yoloOnTheBattlefield) [inline src/components/AssetsListRow/AssetsListRow.js:101]: why do you need back ticks?

File: src/components/AssetsListRow/AssetsListRow.js [hunk 2]
```diff
@@ -121,14 +125,26 @@ export class AssetsListRow extends Component {
...
-                <GreenText price={marketData?.price_change_24h.toFixed(2)}>
-                  {directionIndicator(marketData?.price_change_24h)}$
-                  {marketData?.price_change_24h.toFixed(2)}
+                <GreenText
+                  price={marketData?.price_change_24h_in_currency?.[
+                    currency
+                  ].toFixed(2)}
+                >
+                  {directionIndicator(
+                    marketData?.price_change_24h_in_currency?.[currency]
```
Comment (yoloOnTheBattlefield) [inline src/components/AssetsListRow/AssetsListRow.js:142]: you are using `marketData?.price_change_24h_in_currency?.[currency]` 4 times, better extract this in a variable

File: src/components/AssetsListRow/AssetsListRow.js [hunk 3]
```diff
@@ -169,22 +185,26 @@ export class AssetsListRow extends Component {
...
+                  }
                 >
                   {directionIndicator(
-                    marketData?.current_price.usd - this.state.priceData
+                    marketData?.current_price?.[currency] - this.state.priceData
                   )}
                   {currencyFormat(
-                    marketData?.current_price.usd - this.state.priceData,
-                    '$'
+                    marketData?.current_price?.[currency] -
```
Comment (yoloOnTheBattlefield) [inline src/components/AssetsListRow/AssetsListRow.js:205]: same here

File: src/pages/Coins/Coins.js [hunk 4]
```diff
@@ -13,19 +13,20 @@ export class Coins extends Component {
...
-    volume24h: []
+    volume24h: [],
+    currency: this.props.currency
   };
 
   formatData = (data) => data.map(([x, y]) => ({ x, y: y.toFixed(2) }));
 
-  getChartData = async () => {
+  getChartData = async (currency) => {
     this.setState({ ...this.state, isLoading: true });
```
Comment (yoloOnTheBattlefield) [inline src/pages/Coins/Coins.js:23]: again the spread...

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/AssetsListRow/AssetsListRow.js (hunk 1 above)
Comment (cristhianbenitez) [reply src/components/AssetsListRow/AssetsListRow.js:101]: marketData?.price_change_24h_in_currency?.[currency]  when i'm using the [currency], I'm basically getting the data from the api with the naming of whatever value currency it's.   eg:  marketData?.price_change_24h_in_currency?.['usd']

**Pull request merged**